Examples/Resources/Solvers/PbO-CCSAT-Generic/sparkle_solver_wrapper.py

```diff
@@ -32,9 +32,20 @@
 
 # Construct call from args dictionary
 params = []
-for key in args:
-    if args[key] is not None:
-        params.extend(["-" + str(key), str(args[key])])
+if "config_path" in args:
```

Comment:
This code block looks quite complicated. Is the user required to copy this themselves into their own wrapper and is this a repetative thing? If so I suggest to make a wrapper helper in the sparkle package which users can call to obtain the configuration string, like
```
import sparkle.wrapper_help as wrapper

config = wrapper.get_config(args, as_list=True, keyprefix="-")
```

Reply:
Agreed, will write a ticket